src/main/java/internet/store/Application.java

```diff
@@ -12,10 +17,11 @@
     public static void main(String[] args) {
         ProductService productService = (ProductService) injector.getInstance(ProductService.class);
```

Comment:
Probably it is better to make service classes instances (productService, userService etc.) class variables.

Reply:
Did you mean - final? might be  but not sure...

Reply:
I mean move them outside of main method but I'm not sure as well)